mscore/editdrumset.cpp

```diff
@@ -111,9 +113,46 @@ EditDrumset::EditDrumset(const Drumset* ds, QWidget* parent)
       pitchList->setColumnWidth(1, 60);
       pitchList->setColumnWidth(2, 30);
 
+      QComboBox* combos[] = { wholeCmb, halfCmb, quarterCmb, doubleWholeCmb };
+      for (QComboBox* combo : combos) {
+            // TODO replace smuflRanges by a list of selected symbols
```

Comment:
@danieljray Could you provide a full list of notehead glyphs from smufl that would make sense for drums. Not only for VDL but more globally? Right now, I used every single one in the noteheads range http://www.smufl.org/version/latest/range/noteheads/ which is too much (parenthesis etc...) but not enough since http://www.smufl.org/version/latest/range/roundAndSquareNoteheads/ and http://www.smufl.org/version/latest/range/slashNoteheads/ are not included... So we will need to hardcode a list...

Reply:
I recommend to add the following, currently missing from 3.0:

noteheadXOrnate
noteheadTriangleRightBlack
noteheadCircleSlash
noteheadRoundWhiteWithDot
noteheadLargeArrowUpBlack